<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCSettingsWidget</class>
 <widget class="QDialog" name="QGCSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <attribute name="toolTip">
       <string>General Settings</string>
      </attribute>
      <layout class="QGridLayout" name="generalPaneGridLayout">
       <item row="2" column="0">
        <widget class="QCheckBox" name="lowPowerCheckBox">
         <property name="toolTip">
          <string>Lowers all update rates to save battery power</string>
         </property>
         <property name="text">
          <string>Enable low power mode</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="outdoorStyle">
         <property name="text">
          <string>Use outdoor mission style (light background)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="audioMuteCheckBox">
         <property name="text">
          <string>Mute all audio output</string>
         </property>
         <property name="icon">
          <iconset resource="../../qgroundcontrol.qrc">
           <normaloff>:/files/images/status/audio-volume-muted.svg</normaloff>:/files/images/status/audio-volume-muted.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QRadioButton" name="indoorStyle">
         <property name="text">
          <string>Use indoor mission style (black background)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="titleBarCheckBox">
         <property name="text">
          <string>Show Docked Widget title bars when NOT in advanced Mode.</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="reconnectCheckBox">
         <property name="text">
          <string>Automatically reconnect last link on application startup</string>
         </property>
         <property name="icon">
          <iconset resource="../../qgroundcontrol.qrc">
           <normaloff>:/files/images/devices/network-wireless.svg</normaloff>:/files/images/devices/network-wireless.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="nativeStyle">
         <property name="text">
          <string>Use native platform look and feel (Windows/Linux/Mac OS)</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Log output directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="logDirEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="logDirBrowseButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qgroundcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
